### ISSUE TITLE
invalid module name message cannot be suppressed

### ISSUE THREAD

**Issue by eric-s-raymond** [PyCQA/pylint#5714]
### Bug description

Invalid module-name message can't be suppressed.

### Command used

```shell
pylint --disable=C0103 git-weave
```

### Pylint output

```shell
make pylint
************* Module git-weave
git-weave:1:0: C0103: Module name "git-weave" doesn't conform to snake_case naming style (invalid-name)
git-weave:386:52: W0631: Using possibly undefined loop variable 'line' (undefined-loop-variable)
0
```

### Expected behavior

I expected the first message to be suppressed.

### Pylint version

```shell
pylint 2.7.1
astroid 2.5
Python 3.9.5 (default, Nov 18 2021, 16:00:48)
```

**Comment by DanielNoord** [comment]
Closing as duplicate of #3973
